test/build_and_run_fuzz_targets.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -gt 0 ]]; then
+  FUZZ_TARGETS=$*
+else
+  echo "This script should be called from tools/run_envoy_bazel_coverage.sh"
+fi
+
+LIBFUZZER_TARGETS=""
+# Build all fuzz targets to run instrumented with libfuzzer in sequence.
+for t in ${FUZZ_TARGETS}
+do
+  LIBFUZZER_TARGETS+=${t}_with_libfuzzer
+done
+
+bazel build ${LIBFUZZER_TARGETS} --config asan-fuzzer -c opt
+
+# Now run each fuzz target in parallel for 60 seconds.
+pids=""
+TEMP_CORPORA=""
+
+for t in ${FUZZ_TARGETS}
+do
+  # Make a temporary corpus for this fuzz target.
+  TARGET_BINARY="${t/://}"
+  TEMP_CORPUS_PATH=${TARGET_BINARY:2}
+  CORPUS_DIR="/tmp/${TEMP_CORPUS_PATH////_}_corpus"
```

Comment:
Here you can use `mkdtemp` but in `/tmp`, just to make sure you don't see conflicts if someone runs on the same machine.

Reply:
I suggest creating a single root level dir in `/tmp` with `mkdtemp`.